Logging/.CSProject/anvil-csharp-logging.sln

```diff
@@ -1,12 +1,13 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Anvil.CSharp.Logging", "Anvil.CSharp.Logging.csproj", "{C334863D-1273-44DA-AE08-E006B848D84C}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "anvil-csharp-logging", "anvil-csharp-logging.csproj", "{C334863D-1273-44DA-AE08-E006B848D84C}"
```

Comment:
Does the "Anvil > Logging > Open C# Project" menu item in Unity still work after this?

Reply:
It will when this PR lands https://github.com/decline-cookies/anvil-unity-core/pull/70
(you're too fast 😄 )